src/components/venues/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useContext, useState, useEffect } from "react";
-import { User as FUser } from "firebase";
+import { User as FUser, UserInfo } from "firebase";
```

Comment:
Should `UserInfo` replace `FUser` everywhere?

Reply:
If so we can delete from here

Reply:
yes. Will do on my typescript refactor ticket.